internal/turso/organizations.go

```diff
@@ -161,3 +173,24 @@ func (c *OrganizationsClient) MembersURL(suffix string) (string, error) {
 	}
 	return "/v1/organizations/" + c.client.org + "/members" + suffix, nil
 }
+
+func unsetOrganization() error {
+	settings, err := settings.ReadSettings()
+	if err != nil {
+		return err
+	}
+	return settings.SetOrganization("")
+}
+
+func isNotMemberErr(status int, org string) bool {
+	if status == http.StatusForbidden && org != "" && unsetOrganization() == nil {
+		return true
+	}
+	return false
+}
+
+func notMemberErr(org string) error {
+	msg := fmt.Sprintf("you are not part of organization %s anymore. ", internal.Emph(org))
```

Comment:
The "anymore" word suggests that the user once was a member but is no longer.  I'm not so sure that's always going to be the case.  If not, just remove the word and plainly state lack of membership.